src/composables/getMedicine.js

```diff
@@ -1,7 +1,7 @@
 import medicines from '../apiData/medicines.json';
 
 const getMedicineArray = () => {
-    medicines.map((element) => ({
+    return medicines.map((element) => ({
```

Comment:
``
```suggestion
    const getMedineArray = () => medicines.map( () => ({....}) )
```

Reply:
you have to do something like this:
`const getMedicineArray = () => (medicines.map((element) => ({
        medicineName: element.medicinesName,
        medicinePower: element.power,
        form: element.medicineForm,
        substance: element.medicineSubstance,
        manfacturerName: element.medicineManufacturName,
        leaflet: element.medicineManufacturName,
    })));`